src/test/java/test/io/ticktok/client/support/ServerStub.java

```diff
@@ -28,28 +33,34 @@
     private Channel channel;
 
 
-    public ServerStub(int port, boolean validResponse) throws Exception {
-        createQueue();
+    public ServerStub(int port) throws Exception {
+        createConnection();
         app = new BlinkServer(port) {{
             post("/api/v1/clocks", (req, res) -> {
-                lastClockRequest = new Gson().fromJson(req.body(), ClockRequest.class);
+                ClockRequest clockRequest = new Gson().fromJson(req.body(), ClockRequest.class);
                 try {
                     validateToken(req.param("access_token"));
-                    validateSchedule(lastClockRequest.getSchedule());
+                    validateSchedule(clockRequest.getSchedule());
                 } catch(StatusCodeException e) {
                     res.status(e.getStatus());
                     return "";
                 }
+                final Clock clock = clockFrom(clockRequest);
+                try {
+                    channel.queueDeclare(clock.getChannel().getDetails().get(QUEUE_PARAM), false, false, true, null);
+                } catch (IOException e) {
+                    throw new RuntimeException("Failed to create queue", e);
```

Comment:
why throw a RE ? should'nt we throw some kind of ticktok exception

Reply:
This a test support class, not production code and it's only meant to wrap a checked exception 
if I could I wouldn't catch it at all